exercises/diamond/Diamond.cs

```diff
@@ -1,9 +1,12 @@
 ﻿using System;
 
-public static class Diamond
+namespace Diamond
```

Comment:
Could you remove the namespace? We don't use namespaces in any of the exercises to make things as simple and clean as possible.